core/src/com/unciv/logic/multiplayer/storage/SimpleHttp.kt

```diff
@@ -46,7 +46,8 @@ object SimpleHttp {
                     outputStream.flush()
                 }
 
-                val text = BufferedReader(InputStreamReader(inputStream)).readText()
+                val redirected = responseCode == 301 || responseCode == 302 || responseCode == 303
```

Comment:
I get responseCode 404, so make sure you have the latest version for the server @touhidurrr